starfish/experiment/experiment.py

```diff
@@ -89,11 +118,70 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self):
+        """
+        Describe the aligned subgroups for each Tileset in this FOV
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (8, 2, 1, 205, 405))>',
+             'nuclei': 'Group 0:  <starfish.ImageStack (1, 1, 1, 205, 405))>'}
 
-    def __getitem__(self, item: str) -> ImageStack:
-        return self.get_image(item)
+       Means there are two tilesets in this FOV (primary and nuclei), and because all images have
+       the same (x, y) coordinates, each tileset has a single aligned subgroup.
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (4, 2, 1, 205, 405)>
+                         Group 1:  <starfish.ImageStack (4, 2, 1, 205, 405)>'}
+
+        Means there is one tileset in this FOV, (primary), it has two different aligned
```

Comment:
I hypothesize that users may be interested in the physical coordinates of each group.